lib/java-wrapper.ts

```diff
@@ -77,21 +78,31 @@ export async function getCallGraph(
     config.CALL_GRAPH_GENERATOR_URL,
     config.CALL_GRAPH_GENERATOR_CHECKSUM,
   );
-  const entrypoints = await getEntrypoints(targetPath);
+  const entrypoints = await timeIt('getEntrypoints', () =>
```

Comment:
I wonder if it will be helpful for readability to have the function we measure to be defined on the top?
Something like:
```
const getEntrpointsWithTiming = (targetPath: string) => timeIt('getEntrypoints', () =>   getEntrypoints(targetPath));
...

const entrypoints =  await getEntrpointsWithTiming(targetPath);
```
A suggestion, not sure about it... maybe its to complex? :)

Reply:
Yeah, I'd say that would add more surface area when doing a change without having too much benefit by doing so. But if you feel strongly about it, I can do it.

Reply:
Haha maybe this :) https://www.typescriptlang.org/docs/handbook/decorators.html#method-decorators

Reply:
```
A Decorator is a special kind of declaration that can be attached to a class declaration, method, accessor, property, or parameter. Decorators use the form @expression, where expression must evaluate to a function that will be called at runtime with information about the decorated declaration.
```
unfortunately won't work https://www.typescriptlang.org/docs/handbook/decorators.html#decorators